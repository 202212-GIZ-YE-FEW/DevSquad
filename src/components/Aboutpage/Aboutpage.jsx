import Image from "next/image";
import { useTranslation } from "next-i18next";

import TeamCards from "@/components/Aboutpage/Teamcards";

function Aboutpage() {
    const { t } = useTranslation("common");

    const team = [
        {
            name: "Ramiz Ali",
            github: "https://github.com",
            linkedin: "https://www.linkedin.com",
            instagram: "https://www.instagram.com",
            imageLink: "/images/icon.png",
        },
        {
            name: "Ruqaiah Saif",
            github: "https://github.com/RuqaiahSaif",
            linkedin: "https://www.linkedin.com/in/ruqaiah-saif-086b8a24b/",
            instagram: "https://www.instagram.com",
            imageLink: "/images/icon.png",
        },
        {
            name: " Rawan Salem Bamhdi",
            github: "https://github.com/rrawann",
            linkedin: "https://www.linkedin.com/in/rawan-salem-b-02570521b",
            instagram: "https://instagram.com/rrawnq",
            imageLink: "/images/icon.png",
        },
        {
            name: "Fatima Jubary",
            github: "https://github.com/fatGubari",
            linkedin: "https://www.linkedin.com/in/fatima-abdullah-37248226b/",
            instagram: "https://www.instagram.com",
            imageLink: "/images/icon.png",
        },
        {
            name: "Hekmat Mohammed Ali",
            github: "https://github.com/",
            linkedin: "https://www.linkedin.com/",
            instagram: "https://www.instagram.com/",
            imageLink: "/images/icon.png",
        },
    ];

    return (
        <div>
            <div className='relative'>
                <Image
                    src='/images/aboutPage.png'
                    alt='About Page Image'
                    width={100}
                    height={500}
                    className='w-full '
                    layout='responsive'
                />

                <div className='absolute top-0 left-0 right-0 bottom-0'>
                    <div className='flex h-full w-full flex-col font-Rubik items-center justify-center'>
                        <h3 className='m-1 text-center text-xl font-bold md:m-4 md:text-5xl md:font-bold lg:text-5xl lg:font-bold'>
                            {t("aboutPage.title")}
                        </h3>

                        <p className='text-md mr-12 ml-12 text-center md:ml-24 md:mr-24 md:text-xl lg:ml-32 lg:mr-32 lg:text-2xl'>
                            {t("aboutPage.describtion")}
                        </p>
                    </div>
                </div>
            </div>
            <div className='m-10 min-h-[320px] font-Rubik overflow-hidden rounded-3xl bg-[#fef1e6] md:m-30 md:min-h-[900px] lg:m-44'>
                <h2 className='p-4 text-center text-2xl font-medium md:p-8 md:text-4xl md:font-bold lg:p-12 lg:text-5xl lg:font-bold'>
                    {t("aboutPage.team")}
                </h2>
                <div className='flex flex-row flex-wrap justify-center'>
                    {team.map((member, index) => {
                        return (
                            <TeamCards
                                key={index}
                                name={member.name}
                                github={member.github}
                                linkedin={member.linkedin}
                                instagram={member.instagram}
                                imageLink={member.imageLink}
                            />
                        );
                    })}
                </div>
            </div>
            <div className='m-16 md:m-36'>
                <h2 className='mb-2 text-center font-Rubik text-2xl font-bold md:mb-4 md:text-4xl lg:mb-4 lg:text-4xl'>
                    {t("aboutPage.techTitle")}
                </h2>
                <p className='text-center font-Rubik text-sm md:text-2xl'>
                    {t("aboutPage.techDesc")}
                </p>
            </div>
        </div>
    );
}

export default Aboutpage;
