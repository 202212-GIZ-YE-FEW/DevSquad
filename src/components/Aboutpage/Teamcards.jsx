import AOS from "aos";
import Image from "next/image";
import { useEffect } from "react";

import "aos/dist/aos.css";
const Teamcards = ({ name, github, linkedin, instagram, imageLink }) => {
    useEffect(() => {
        AOS.init();
    }, []);
    return (
        <div>
            <div className='m-4 h-28 w-28 overflow-hidden rounded-2xl bg-primary-lightblue md:m-8 md:h-60 md:w-60 lg:m-12 lg:h-80 lg:w-80 transition duration-500 hover:scale-105'>
                <div className='mt-2 flex justify-center md:mt-3 lg:mt-3'>
                    <div className='relative h-12 w-12 overflow-hidden rounded-full pt-2 text-center md:mt-4 md:h-28 md:w-28 md:pt-4 lg:h-36 lg:w-36'>
                        <Image
                            src={imageLink}
                            alt='User'
                            // width={200}
                            // height={150}
                            className='w-full w-100% h-100'
                            objectFit='cover'
                            layout='fill'
                        />
                    </div>
                </div>
                <h4 className='pt-2 pb-2 text-center text-[10px] font-medium md:pb-2 md:pt-2 md:text-[24px]'>
                    {name}
                </h4>
                <div className='align-start content-bottom flex flex-row justify-center pb-2 md:ml-1 md:mt-2 md:content-end lg:mt-8'>
                    <div className='m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12'>
                        <a href={linkedin} target='_blank'>
                            <Image
                                src='/images/linkedin.png'
                                alt='User'
                                width={100}
                                height={100}
                            />
                        </a>
                    </div>
                    <div className='m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12'>
                        <a href={github} target='_blank'>
                            <Image
                                src='/images/github.png'
                                alt='User'
                                width={100}
                                height={100}
                            />
                        </a>
                    </div>
                    <div className='m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12'>
                        <a href={instagram} target='_blank'>
                            <Image
                                src='/images/instagram.png'
                                alt='User'
                                width={100}
                                height={100}
                            />
                        </a>
                    </div>
                </div>
            </div>
        </div>
    );
};

export default Teamcards;
