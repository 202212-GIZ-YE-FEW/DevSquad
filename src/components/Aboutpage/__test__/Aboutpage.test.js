import renderer from "react-test-renderer";

import Aboutpage from "../Aboutpage";

it("renders correctly", () => {
    const tree = renderer.create(<Aboutpage />).toJSON();
    expect(tree).toMatchSnapshot();
});
