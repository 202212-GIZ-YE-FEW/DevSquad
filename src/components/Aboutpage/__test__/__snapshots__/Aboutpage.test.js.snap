// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="relative"
  >
    <img
      alt="About Page Image"
      className="w-full "
      data-nimg="1"
      decoding="async"
      height={500}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      sizes="100vw"
      src="/_next/image?url=%2Fimages%2FaboutPage.png&w=3840&q=75"
      srcSet="/_next/image?url=%2Fimages%2FaboutPage.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2FaboutPage.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2FaboutPage.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2FaboutPage.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FaboutPage.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FaboutPage.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FaboutPage.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FaboutPage.png&w=3840&q=75 3840w"
      style={
        Object {
          "color": "transparent",
          "height": "auto",
          "width": "100%",
        }
      }
      width={100}
    />
    <div
      className="absolute top-0 left-0 right-0 bottom-0"
    >
      <div
        className="flex h-full w-full flex-col font-Rubik items-center justify-center"
      >
        <h3
          className="m-1 text-center text-3xl font-bold md:m-4 md:text-5xl md:font-bold lg:text-5xl lg:font-bold"
        >
          aboutPage.title
        </h3>
        <p
          className="text-md mr-12 ml-12 text-center md:ml-24 md:mr-24 md:text-2xl lg:ml-32 lg:mr-32 lg:text-2xl"
        >
          aboutPage.describtion
        </p>
      </div>
    </div>
  </div>
  <div
    className="m-10 min-h-[320px] font-Rubik overflow-hidden rounded-3xl bg-[#fef1e6] md:m-30 md:min-h-[900px] lg:m-44"
  >
    <h2
      className="p-4 text-center text-2xl font-medium md:p-8 md:text-4xl md:font-bold lg:p-12 lg:text-5xl lg:font-bold"
    >
      aboutPage.team
    </h2>
    <div
      className="flex flex-row flex-wrap justify-center"
    >
      <div>
        <div
          className="m-4 h-28 w-28 overflow-hidden rounded-2xl bg-primary-lightblue md:m-8 md:h-60 md:w-60 lg:m-12 lg:h-80 lg:w-80"
        >
          <div
            className="mt-2 flex justify-center md:mt-3 lg:mt-3"
          >
            <div
              className="relative h-12 w-12 overflow-hidden rounded-full pt-2 text-center md:mt-4 md:h-28 md:w-28 md:pt-4 lg:h-36 lg:w-36"
            >
              <img
                alt="User"
                className="w-full w-100% h-100"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                onError={[Function]}
                onLoad={[Function]}
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75"
                srcSet="/_next/image?url=%2Fimages%2Ficon.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Ficon.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Ficon.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Ficon.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Ficon.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Ficon.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Ficon.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75 3840w"
                style={
                  Object {
                    "bottom": 0,
                    "color": "transparent",
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": "100%",
                  }
                }
              />
            </div>
          </div>
          <h4
            className="pt-2 pb-2 text-center text-[10px] font-medium md:pb-2 md:pt-2 md:text-[24px]"
          >
            Ramiz Ali
          </h4>
          <div
            className="align-start content-bottom flex flex-row justify-center pb-2 md:ml-1 md:mt-2 md:content-end lg:mt-8"
          >
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.linkedin.com"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Flinkedin.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://github.com"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fgithub.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.instagram.com"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Finstagram.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="m-4 h-28 w-28 overflow-hidden rounded-2xl bg-primary-lightblue md:m-8 md:h-60 md:w-60 lg:m-12 lg:h-80 lg:w-80"
        >
          <div
            className="mt-2 flex justify-center md:mt-3 lg:mt-3"
          >
            <div
              className="relative h-12 w-12 overflow-hidden rounded-full pt-2 text-center md:mt-4 md:h-28 md:w-28 md:pt-4 lg:h-36 lg:w-36"
            >
              <img
                alt="User"
                className="w-full w-100% h-100"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                onError={[Function]}
                onLoad={[Function]}
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75"
                srcSet="/_next/image?url=%2Fimages%2Ficon.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Ficon.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Ficon.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Ficon.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Ficon.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Ficon.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Ficon.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75 3840w"
                style={
                  Object {
                    "bottom": 0,
                    "color": "transparent",
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": "100%",
                  }
                }
              />
            </div>
          </div>
          <h4
            className="pt-2 pb-2 text-center text-[10px] font-medium md:pb-2 md:pt-2 md:text-[24px]"
          >
            Ruqaiah Saif
          </h4>
          <div
            className="align-start content-bottom flex flex-row justify-center pb-2 md:ml-1 md:mt-2 md:content-end lg:mt-8"
          >
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.linkedin.com/in/ruqaiah-saif-086b8a24b/"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Flinkedin.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://github.com/RuqaiahSaif"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fgithub.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.instagram.com"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Finstagram.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="m-4 h-28 w-28 overflow-hidden rounded-2xl bg-primary-lightblue md:m-8 md:h-60 md:w-60 lg:m-12 lg:h-80 lg:w-80"
        >
          <div
            className="mt-2 flex justify-center md:mt-3 lg:mt-3"
          >
            <div
              className="relative h-12 w-12 overflow-hidden rounded-full pt-2 text-center md:mt-4 md:h-28 md:w-28 md:pt-4 lg:h-36 lg:w-36"
            >
              <img
                alt="User"
                className="w-full w-100% h-100"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                onError={[Function]}
                onLoad={[Function]}
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75"
                srcSet="/_next/image?url=%2Fimages%2Ficon.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Ficon.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Ficon.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Ficon.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Ficon.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Ficon.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Ficon.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75 3840w"
                style={
                  Object {
                    "bottom": 0,
                    "color": "transparent",
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": "100%",
                  }
                }
              />
            </div>
          </div>
          <h4
            className="pt-2 pb-2 text-center text-[10px] font-medium md:pb-2 md:pt-2 md:text-[24px]"
          >
             Rawan Salem Bamhdi
          </h4>
          <div
            className="align-start content-bottom flex flex-row justify-center pb-2 md:ml-1 md:mt-2 md:content-end lg:mt-8"
          >
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.linkedin.com/in/rawan-salem-b-02570521b"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Flinkedin.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://github.com/rrawann"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fgithub.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://instagram.com/rrawnq"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Finstagram.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="m-4 h-28 w-28 overflow-hidden rounded-2xl bg-primary-lightblue md:m-8 md:h-60 md:w-60 lg:m-12 lg:h-80 lg:w-80"
        >
          <div
            className="mt-2 flex justify-center md:mt-3 lg:mt-3"
          >
            <div
              className="relative h-12 w-12 overflow-hidden rounded-full pt-2 text-center md:mt-4 md:h-28 md:w-28 md:pt-4 lg:h-36 lg:w-36"
            >
              <img
                alt="User"
                className="w-full w-100% h-100"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                onError={[Function]}
                onLoad={[Function]}
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75"
                srcSet="/_next/image?url=%2Fimages%2Ficon.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Ficon.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Ficon.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Ficon.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Ficon.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Ficon.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Ficon.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75 3840w"
                style={
                  Object {
                    "bottom": 0,
                    "color": "transparent",
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": "100%",
                  }
                }
              />
            </div>
          </div>
          <h4
            className="pt-2 pb-2 text-center text-[10px] font-medium md:pb-2 md:pt-2 md:text-[24px]"
          >
            Fatima Jubary
          </h4>
          <div
            className="align-start content-bottom flex flex-row justify-center pb-2 md:ml-1 md:mt-2 md:content-end lg:mt-8"
          >
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.linkedin.com/in/fatima-abdullah-37248226b/"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Flinkedin.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://github.com/fatGubari"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fgithub.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.instagram.com"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Finstagram.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
          </div>
        </div>
      </div>
      <div>
        <div
          className="m-4 h-28 w-28 overflow-hidden rounded-2xl bg-primary-lightblue md:m-8 md:h-60 md:w-60 lg:m-12 lg:h-80 lg:w-80"
        >
          <div
            className="mt-2 flex justify-center md:mt-3 lg:mt-3"
          >
            <div
              className="relative h-12 w-12 overflow-hidden rounded-full pt-2 text-center md:mt-4 md:h-28 md:w-28 md:pt-4 lg:h-36 lg:w-36"
            >
              <img
                alt="User"
                className="w-full w-100% h-100"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                onError={[Function]}
                onLoad={[Function]}
                sizes="100vw"
                src="/_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75"
                srcSet="/_next/image?url=%2Fimages%2Ficon.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Ficon.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Ficon.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Ficon.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Ficon.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Ficon.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Ficon.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Ficon.png&w=3840&q=75 3840w"
                style={
                  Object {
                    "bottom": 0,
                    "color": "transparent",
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": undefined,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": "100%",
                  }
                }
              />
            </div>
          </div>
          <h4
            className="pt-2 pb-2 text-center text-[10px] font-medium md:pb-2 md:pt-2 md:text-[24px]"
          >
            Hekmat Mohammed Ali
          </h4>
          <div
            className="align-start content-bottom flex flex-row justify-center pb-2 md:ml-1 md:mt-2 md:content-end lg:mt-8"
          >
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.linkedin.com/"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Flinkedin.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Flinkedin.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://github.com/"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fgithub.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
            <div
              className="m-px h-4 w-4 hover:cursor-pointer md:h-8 md:w-8 lg:h-12 lg:w-12"
            >
              <a
                href="https://www.instagram.com/"
                target="_blank"
              >
                <img
                  alt="User"
                  data-nimg="1"
                  decoding="async"
                  height={100}
                  loading="lazy"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="/_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Finstagram.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Finstagram.png&w=256&q=75 2x"
                  style={
                    Object {
                      "color": "transparent",
                    }
                  }
                  width={100}
                />
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="m-16 md:m-36"
  >
    <h2
      className="mb-2 text-center font-Rubik text-2xl font-bold md:mb-4 md:text-4xl lg:mb-4 lg:text-4xl"
    >
      aboutPage.techTitle
    </h2>
    <p
      className="text-center font-Rubik text-sm md:text-2xl"
    >
      aboutPage.techDesc
    </p>
  </div>
</div>
`;
